hooks/post_gen_project.py

```diff
@@ -81,8 +81,8 @@ def set_flag(file_path, flag, value=None):
         f.truncate()
 
 
-def set_django_secret_key(file_path):
-    return set_flag(file_path, "!!!DJANGO_SECRET_KEY!!!")
+def set_django_secret_key(file_path, value=None):
+    return set_flag(file_path, "!!!DJANGO_SECRET_KEY!!!", value=value)
```

Comment:
Instead of `value=None` as a kwarg, maybe it would be "safer" to make `value` a positional argument, because I don't think we ever want `value` to be `None`. That would help prevent misuse of this function.

Reply:
I did that here so that we do not have to change code that is already using this function, but you are right that would be a mistake if the values did not match up in all 3 areas!